test/e2e-secondary-network/infra/prepare_cluster_common.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+# Copyright 2022 Antrea Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+function generateAntreaConfig() {
+        genManifest=$THIS_DIR/../../../../hack/generate-manifest.sh
+	if [ "$1" == "sriov" ]; then
+        	if [ "$sriov_interface" != '' ] && [ "$vfs_num" != '' ]; then
+                	$genManifest --sriov --whereabouts > $YAML_ANTREA
+		fi
+	
+	elif [ "$1" == "veth" ]; then
+                $genManifest --whereabouts > $YAML_ANTREA
+        fi
+
+        if [ $? -ne 0 ]; then
+              echoerr "Failed to generate Antrea.yml"
+              exit 1
+        fi
+        echo "Antrea.yml generated with required secondary network configuration parameters."
+	sed -i 's/#  \([[:space:]]*SecondaryNetwork.*\)/\1/' $YAML_ANTREA 
+	if [ $? -ne 0 ]; then
+              echoerr "Failed to update Antrea"
+              exit 1
+        fi
+	sleep 2
+	sed -i '/^\([[:space:]]*#SecondaryNetwork: *\).*/s## true#g' $YAML_ANTREA
+}
+#ARUN - common
+function updateKubeConfigForTargetCluster() {
+        mkdir -p /tmp/kubernetes/$hostIP
+        scp $hostIP:/etc/kubernetes/admin.conf /tmp/kubernetes/$hostIP/.
+        if [ $? -ne 0 ]; then
+                echoerr "Failed to copy the downloaded configuration files!"
+                exit 1
+        fi
+        echo export KUBECONFIG="/tmp/kubernetes/$hostIP/admin.conf" > env.sh
+        source ./env.sh
+        #echo "kubeconfig updated with the remote cluster configuration!"
```

Comment:
remove commented out lines that you no longer need

Reply:
removed.